src/testHelpers.ts

```diff
@@ -40,11 +40,16 @@ export function initTestHelpers() {
 
   function setup() {
     if (isJSDOMEnvironment()) {
-      // Remove initial JSDOM <head> element
-      const headElement = document.querySelector('head');
-      if (headElement) {
-        headElement.remove();
```

Comment:
This was the reason that `_app` import did not resolve correctly. At import time construction of `styletron's Client` failed because of this line: https://github.com/styletron/styletron/blob/master/packages/styletron-engine-atomic/src/client/client.js#L209